### DISCUSSION TITLE
Removed official support for direct MSL

### DISCUSSION THREAD

**Discussion by Jbudone** [KhronosGroup/MoltenVK#1314] [General]
I was hoping there'd be some reasoning explained behind this change `commit 612c84d79eb080e1ec6ee018f45f17dcd63205ae`

Why is official support removed? Surely this will have a large performance impact for applications with many complex shaders that need to be loaded on the fly

**Comment by cdavis5e** [comment]
The short answer is that it bit-rotted. See #956 and #1253 for some more details.
